packages/sku/config/eslint/eslintConfig.js

```diff
@@ -1,9 +1,23 @@
 const { eslintDecorator } = require('../../context');
+const importOrderConfig = require('./importOrderConfig');
+const coreConfig = require('eslint-config-seek');
 
-const coreConfig = require.resolve('eslint-config-seek');
-
-const baseConfig = {
-  extends: [coreConfig, require.resolve('./importOrderConfig')],
-};
+const baseConfig = [
+  {
+    ignores: [
```

Comment:
Worth considering if you migrate this on behalf of users (any custom setting they have) in some form, or just say "use dangerouslySetESLintConfig" or similar